healthcare/api-client/v1/dicom/dicomweb.py

```diff
@@ -255,21 +402,47 @@ def dicomweb_retrieve_rendered(
 
 
 # [START healthcare_dicomweb_delete_study]
-def dicomweb_delete_study(
-    base_url, project_id, cloud_region, dataset_id, dicom_store_id, study_uid
-):
+def dicomweb_delete_study(project_id, location, dataset_id, dicom_store_id, study_uid):
     """Handles DELETE requests equivalent to the GET requests specified in
     the WADO-RS standard.
-    """
-    url = "{}/projects/{}/locations/{}".format(base_url, project_id, cloud_region)
+
+    See https://github.com/GoogleCloudPlatform/python-docs-samples/tree/master/healthcare/api-client/v1/dicom
+    before running the sample."""
+    # Imports Python's built-in "os" module
+    import os
+
+    # Imports the google.auth.transport.requests transport
+    from google.auth.transport import requests
+
+    # Imports a module to allow authentication using a service account
+    from google.oauth2 import service_account
+
+    # Gets credentials from the environment.
+    credentials = service_account.Credentials.from_service_account_file(
+        os.environ["GOOGLE_APPLICATION_CREDENTIALS"]
+    )
+    scoped_credentials = credentials.with_scopes(
+        ["https://www.googleapis.com/auth/cloud-platform"]
+    )
+    # Creates a requests Session object with the credentials.
+    session = requests.AuthorizedSession(scoped_credentials)
+
+    # URL to the Cloud Healthcare API endpoint and version
+    base_url = "https://healthcare.googleapis.com/v1"
+
+    # TODO(developer): Uncomment these lines and replace with your values.
+    # project_id = 'my-project'  # replace with your GCP project ID
+    # location = 'us-central1'  # replace with the parent dataset's location
+    # dataset_id = 'my-dataset'  # replace with the parent dataset's ID
+    # dicom_store_id = 'my-dicom-store' # replace with the DICOM store ID
+    # study_uid = '1.3.6.1.4.1.5062.55.1.2270943358.716200484.1363785608958.61.0'  # replace with the study UID
```

Comment:
If these fields are passed on from the arguments, does the developer need to edit them again? Do we expect developers to either edit these values or pass them down as arguments? Or are they present for the sake of region tags?

Reply:
They are present because of our guidelines to make code samples copy+pastable from docs (with some explanation of the sample values the user needs to pass in) without introducing a regression where the samples no longer work as a CLI application. It's not ideal in that it presents the user with two options, but it I think it's a good compromise. Unless we want to make these samples purely copy+pastable from the docs without being able to run as a CLI, in which case we'd just remove the function arguments.

Reply:
Sounds good. Thanks for explaining it! :)